package/MDAnalysis/analysis/dssp/dssp.py

```diff
@@ -0,0 +1,408 @@
+"""
+Secondary structure assignment (helix, sheet and loop) --- :mod:`MDAnalysis.analysis.dssp`
+==========================================================================================
+
+:Author: Egor Marin
+:Year: 2024
+:Copyright: LGPL v2.1+
+
+.. versionadded:: 2.8.0
+
+The module contains code to build hydrogend bond contact map,
+and use it to assign protein secondary structure (:class:`DSSP`).
+
+This module uses the python version of the original algorithm :footcite:p:`Kabsch1983`,
+re-implemented by @ShintaroMinami and available under MIT license from
+`ShintaroMinami/PyDSSP <https://github.com/ShintaroMinami/PyDSSP/tree/master#differences-from-the-original-dssp>`_.
+
+
+.. Note::
+    This implementation does not discriminate different types of
+    beta-sheets, as well as different types of helices, meaning you will get
+    :math:`3_{10}` helices and π-helices labelled as "helix" too.
+
+
+Using original `pydssp`
```

Comment:
add un-intrusive structure
```suggestion
.. rubric:: Using original `pydssp`
```